src/ui/components/pages/assets/assetItem.tsx

```diff
@@ -174,7 +175,7 @@ export function AssetItem({
           </Tooltip>
 
           {currentNetwork === 'mainnet' &&
-            isSwappableAsset(currentNetwork, assetId) && (
+            Object.values(swappableAssets).flat().includes(assetId) && (
```

Comment:
May be pass `isSwappable` from the outside? Otherwise for every asset item there's a loop, which first flattens arrays and then searches for asset id. Also when passing from outside (asset list) it would make sense to convert it to set first, to utilize `Set#has` method, which I beleive is `O(1)`, not `O(n)`